llama-index-integrations/selectors/llama-index-selectors-notdiamond/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = true
+import_path = "llama_index.selectors.notdiamond"
```

Comment:
We can also add the `[tool.llamahub.class_authors]` section that other integrations have. This helps the integration show up on llamahub

Reply:
Thanks for noting - I've added it just below this block.